src/events/sql.js

```diff
@@ -104,7 +104,10 @@ module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callbac
                         rows = res;
                     }
                     eventInterface.addToMetadata(dbapiEvent, { rowCount });
-                    if (rowCount && rows instanceof Array && rows.length) {
+                    if (rowCount &&
+                        rows instanceof Array &&
+                        rows.length &&
+                        !tracer.doesContainIgnoredKey(config.getConfig().ignoredDBTables, table)) {
```

Comment:
you've created a very big change just to check if `config.getConfig().ignoredDBTables.includes(table)`. Can you undo all changes including the packge-lock.json and simplify this PR please